data/rules/rules_medicine2.yml

```diff
@@ -1,25 +1,88 @@
 version: "2.0"
```

Comment:
File umbenennen in rules_medicine.yml ohne die 2, es gibt ja nur noch ein File.

Reply:
erledigt